tests/test_digraph.py

```diff
@@ -23,9 +23,20 @@ def run(self):
 def test_combine_nodes():
     with znflow.DiGraph() as graph:
         n1 = ComputeSum(inputs=(1, 2, 3))
-        n2 = ComputeSum(inputs=(4, 5, 6))
-        n3 = ComputeSum(inputs=(n1.outputs, n2.outputs))
+        n2 = add(4, 5, 6)
```

Comment:
Why did you do this? To check wether different sums still work? If so when would be a case where that is needed?

Reply:
The idea behind the test was to combine different nodes. Here I did only combine to class based nodes but I wanted to add a function based one to make sure, that all combinations work.